web/src/OverviewActionBar.stories.tsx

```diff
@@ -22,11 +23,13 @@ export default {
       history.location.search = `?level=${level}&source=${source}`
       return (
         <Router history={history}>
-          <StarredResourceMemoryProvider>
-            <div style={{ margin: "-1rem", height: "80vh" }}>
-              <Story {...context.args} />
-            </div>
-          </StarredResourceMemoryProvider>
+          <TiltSnackbarProvider>
```

Comment:
fyi, there were a couple instances where the missing snackbar provider caused stories to crash.

Reply:
:(

I think this will be pretty easy to check in CI - https://app.shortcut.com/windmill/story/12729/test-that-storybook-stories-render-without-error

I've been toying with the idea of just making a function to return all (well, most) of the context providers we need, and using that in both HUD and in tests. The indentation in HUD is getting a bit much, and it's pretty annoying to add these all over.

Reply:
both of those are great ideas! maybe i can look into it later (if you are busy with other things!)